src/shared/fixed-periods/fixed-periods.js

```diff
@@ -680,7 +687,7 @@ const getFinancialJulyPeriodType = (formatYyyyMmDd, fnFilter) => {
 }
 
 const getFinancialAprilPeriodType = (formatYyyyMmDd, fnFilter) => {
-    return (config) => {
+    return (config = {}) => {
```

Comment:
Right now, there's no scenario where the object isn't being passed. But all properties either change the control flow and are optional (`filterFuturePeriods` & `reversePeriods`) or have a default value (`offset`, which defaults to `0`).

In a previous solution, I didn't pass anything to `getPeriods` and an error was thrown. Right now I think I should either remove the default or add it to all functions returned by any `get[Name]PeriodType`. Not sure what would be the better solution. I guess removing it is fine as I prefer to create a library for handling our fixed dates so that every app could benefit from that. But I suppose that that library will look completely different as we'll put a lot more thought into it.

What do you think?